### PR TITLE
Bump Deps

### PR DESCRIPTION
**Pull request by nimrod-becker**
### Explain the changes
1. Bump Deps

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,8 +91,8 @@
     "linux-blockutils": "0.1.0",
     "lodash": "4.17.15",
     "mime": "2.4.4",
-    "minimist": "1.2.0",
-    "mkdirp": "0.5.1",
+    "minimist": "1.2.5",
+    "mkdirp": "1.0.4",
```
Comment (guymguym) [inline package.json:95]: mkdirp major version changed from callback to promise, no need for code changes too??
See here - https://github.com/isaacs/node-mkdirp/blob/master/CHANGELOG.md

File: package.json [hunk 2]
```diff
@@ -106,10 +106,10 @@
     "node-df": "0.1.4",
     "node-gyp": "4.0.0",
     "performance-now": "2.1.0",
-    "prom-client": "11.5.3",
+    "prom-client": "12.0.0",
```
Comment (guymguym) [inline package.json:109]: This major version change is safe?

Comment (nimrod-becker) [reply package.json:109]: Asked @jeniawhite tyo check it a while back, yes

**Review by guymguym — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (nimrod-becker) [reply package.json:109]: What's weird is that it seems like either yaml or was breaks travis, and its only patch version

**Pull request merged**